dev-lang/vala/vala-0.36.11.recipe

```diff
@@ -75,18 +98,24 @@ INSTALL()
 {
 	make install
 
-	# remove libtool library files
-	rm $libDir/lib*.la
+	rm -f $libDir/*.la
 
 	fixPkgconfig
 
+	#devel and common packages
+	packageEntries devel \
+		$developDir \
+		$manDir
+
 	packageEntries common \
 		$dataDir/aclocal/vapigen.m4 \
 		$dataDir/aclocal/vala.m4 \
 		$dataDir/vala/Makefile.vapigen
 
 	# Avoid having an empty data/aclocal/ in vala$secondaryArchSuffix
 	rmdir $dataDir/aclocal
+
+	rmdir $documentationDir
```

Comment:
Having it in or not doesn't seem to make any difference?

Reply:
Exact. The resulting package is the same.
That's because when haikuporter sees that the directory is empty (after `INSTALL()` has done its job) it does the `rmdir` for us.

That said, having the `rmdir $documentationDir` (near the end of `INSTALL()`) allows to trigger an error in case upstream decides to add something in a new release.

BTW, each choice has its small advantages and its small downsides, so I don't have any preference :)

Reply:
Looks ok to so far then :)